ci-operator/config/openstack-k8s-operators/openstack-ansibleee-operator/openstack-k8s-operators-openstack-ansibleee-operator-main.yaml

```diff
@@ -51,10 +51,10 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: unit
+- as: functional
   commands: |
     export GOFLAGS=
-    make gotest
+    make test GINKGO_ARGS='--no-color'
```

Comment:
```yaml
- as: functional
  commands: |
    mkdir -p ../operator && cp -r . ../operator
    cd ../operator
    export GOFLAGS=
    make test GINKGO_ARGS='--no-color'
```
otherwise it fails with permission error when it tries to copy the CRD:
```
/go/src/github.com/openstack-k8s-operators/openstack-ansibleee-operator/bin/controller-gen rbac:roleName=manager-role crd:maxDescLen=0 webhook paths="./..." output:crd:artifacts:config=config/crd/bases && \
rm -f api/bases/* && cp -a config/crd/bases api/
cp: preserving times for 'api/bases': Operation not permitted
```
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/39744/rehearse-39744-pull-ci-openstack-k8s-operators-openstack-ansibleee-operator-main-functional/1661770020058828800#1:build-log.txt%3A122

Reply:
I've tweaked the make file in #159 so now that is landed I will re-run this. If it still fails with permission then I will follow your suggestions.

Reply:
The makefile tweak was enough. It is passing now https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/39744/rehearse-39744-pull-ci-openstack-k8s-operators-openstack-ansibleee-operator-main-functional/1663509062714658816/artifacts/test/build-log.txt